olwebsite-app/src/pages/LoginView.jsx

```diff
@@ -1,26 +1,121 @@
-import React from "react";
+import React, { useState } from "react";
+import { Redirect, withRouter } from "react-router-dom";
+
+import { 
+  validateEmail, 
+  validatePassword
+} from '../utils/authentication-utils';
+import { SUCCESS } from '../constants/authentication-constants';
+import { LOGIN_PATH } from '../constants/rest-constants';
 
 import Input from '../components/Objects/Input/Input';
 import homeLogo from "../images/homeLogo.png";
 
-export default function LoginView() {
+function LoginView() {
+  /************************************
+   * State
+   ************************************/
+
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
+  const [errorMessage, setErrorMessage] = useState(null);
+
+  const [email, setEmail] = useState('');
+  const [emailErrorMessage, setEmailErrorMessage] = useState(null);
+  const [password, setPassword] = useState('');
+  const [passwordErrorMessage, setPasswordErrorMessage] = useState(null);
+
+  /************************************
+   * Helper Functions
+   ************************************/
+
+ /* Handler for log in button clicked, if valid inputs logs in user
+  */
+  async function loginClicked() {
+    if (validateInput()) {
+      await loginUser();
+    }
+  }
+
+ /* Makes REST request to log in user, based on response status will 
+  * redirect to the landing view or display and error message
+  */
+  async function loginUser() {
```

Comment:
Sounds good, what would be the reason of having an AbortController? I don't see a simple post request taking that long

Reply:
It might depending on the internet connection speed, or if for whatever reason the user navigates to a different part of the site and causes the component in which the request was submitted to be unmounted before the request is completed. Although, I don't see that being a really common occurrence, I think it's best to have it all ironclad just in case.

Also, hold off from moving the function to a separate file, so we can get the CI PR merged first.

Reply:
Done